### PR TITLE
convert github issue templates to forms

### PR DESCRIPTION
**Pull request by mrbobbytables**
Largely a no-op converting the issue templates to [GitHub Issue Forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms)

Other changes include:
- Optional request in org membership form to take the LF [Inclusive Open Source Community Orientation course](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
- Repo request form now has a checkbox explicitly asking if its a staging repo

Previews of what these forms look like are available here: https://github.com/mrbobbytables/issue-template-test/issues

/hold

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/general-support.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    description: What user is impacted?
+    placeholder: e.g. @example_user
+  validations:
+    required: false
+- id: problem
+  type: textarea
+  attributes:
+    label: Describe the issue
+    placeholder: A clear and concise description of what you need assistance with.
+    render: markdown
```
Comment (spiffxp) [inline .github/ISSUE_TEMPLATE/general-support.yml:25]: This ends up dumping text
```
in a preformatted box that doesn't do word wrapping at all
```
```suggestion
```

**Review by spiffxp — CHANGES_REQUESTED** [review]
One nit, otherwise LGTM

File: .github/ISSUE_TEMPLATE/general-support.yml (hunk 1 above)
Comment (mrbobbytables) [reply .github/ISSUE_TEMPLATE/general-support.yml]: done 👍

**Comment by ameukam** [comment]
/lgtm
/hold
Remove hold when ready

**Pull request merged**